src/style.css

```
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :first-child,
```

```suggestion
.stars--1 :nth-child(-n+1),
```

Make all code the same style.